configuration.yml

```diff
@@ -19,7 +19,7 @@ alluxio_port: "19998"
 pyspark_cassandra_version: "0.3.5"
 analysers_path: "/app/analysers"
 transformers_path: "/app/data-transformers"
-configurations_path: "/app/configuration"
+configurations_path: "/app/data-transformers/suts/wfms"
```

Comment:
In the `benchflow-test.yml` configuration file you also have the `type` of the SUT